pkg/cvo/sync_test.go

```diff
@@ -194,7 +194,7 @@ func Test_SyncWorker_apply(t *testing.T) {
 			}
 			worker.payload = up
 
-			err := worker.apply(ctx, &SyncWork{}, 1, &statusWrapper{w: worker, previousStatus: worker.Status()})
+			err, _ := worker.apply(ctx, &SyncWork{}, 1, &statusWrapper{w: worker, previousStatus: worker.Status()})
```

Comment:
You can drop these changes to avoid [test compilation issues][1], now that we're no longer tweaking the `apply` signature.

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_cluster-version-operator/837/pull-ci-openshift-cluster-version-operator-master-lint/1573415988077006848#1:build-log.txt%3A22